akka-http/build.sbt

```diff
@@ -55,7 +55,6 @@ val `akka-http-server` =
       excludeDependencies ++= {
         if (scalaBinaryVersion.value.startsWith("3")) {
           List(
-            ExclusionRule("org.scala-lang.modules", "scala-xml_3"),
```

Comment:
@julienrf Is this change fine? Tests will not work on Scala 3 with this exclusion.

Reply:
Yes, that’s even better! Eventually, we should get rid of those exclusions.

Reply:
That makes me think that we should probably not publish Scala 3 artifacts for the modules that still use `CrossVersion.for3Use2_13` in their dependencies… :thinking: